scale/job/configuration/configurators.py

```diff
@@ -318,14 +318,15 @@ def _configure_all_tasks(self, config, job_exe, job_type):
         # Configure tasks for logging
         if settings.LOGGING_ADDRESS is not None:
             log_driver = DockerParameter('log-driver', 'syslog')
+            debug_level_parameter = DockerParameter('env','LOGGING_LEVEL=%s' % debug_level)
```

Comment:
Instead of directly adding a Docker parameter here, use an env var. They will be converted correctly to appropriate Docker parameters later in the configurator.

```
logging_env_vars = {'LOGGING_LEVEL': logging_level}
config.add_to_task('pre', env_vars=logging_env_vars, docker_params=[...])
```